helm/flowforge/templates/service-ingress.yaml

```
@@ -47,3 +47,43 @@ spec:
     - {{ $forgeHostname }}
     secretName: {{ $forgeHostname }}
   {{- end }}
+{{- if gt (int .Values.forge.replicas) 1 -}}
```

Objects separator is missing

```suggestion
---
{{- if gt (int .Values.forge.replicas) 1 -}}
```